controllers/datafeed_controller.py

```diff
@@ -595,6 +621,10 @@ def get(self, event_key):
         if type(event_teams) is not list:
             event_teams = [event_teams]
 
+        avatars = df.getEventTeamAvatars(event.year, event.key_name)
```

Comment:
Let's wrap this whole section in a check for `if event.year == 2018:`

Also, the only argument should be `event.key_name`